src/main/webapp/javascript/event.js

```diff
@@ -20,13 +20,39 @@ function loadEvent() {
             document.getElementById("voting").classList.remove("w3-hide");
             document.getElementById("feedback").classList.add("w3-hide");
             getVotingPoll();
-        } else if (event.eventStatus === "CLOSED") {
+        } else if (event.eventStatus === "FINISHED") {
             document.getElementById("voting").classList.add("w3-hide");
-            document.getElementById("feedback").classList.remove("w3-hide");
+            document.getElementById("feedbackInput").classList.remove("w3-hide");
             getFeedbackPoll();
+            getTextFeedback();
         }
     })
 }
+function checkIfConatainsText() {
+    var str1 = document.getElementById("comment").value;
+    if (document.getElementById("comment").value.replace(/\s/g, '').length) {
+        saveTextFeedback()
+    }
+    else {
```

Comment:
In Java and JavaScript, please keep `else {` on the same line as preceding `}`:
`} else {`